dist/logic.js

```diff
@@ -0,0 +1,27 @@
+class ExpenseManager {
+    constructor() {
+        this.expenses = []
+        this.newExpense = {}
+    }
+
+    async addExpense(name,amount,group,date) {
+        const addObjExpenses = {
+            name: name,
+            amount: amount,
+            group: group,
+            date: date
```

Comment:
if the key and the value are the same, use the shorthand: [here](https://alligator.io/js/object-property-shorthand-es6/)